pkg/workflows/wasm/host/wasm_test.go

```diff
@@ -293,7 +234,7 @@ func Test_Compute_Emit(t *testing.T) {
 		assert.Nil(t, err)
 	})
 
-	t.Run("failure on emit writes to error chain and logs", func(t *testing.T) {
+	t.Run("failure on emit does not throw error chain but logs the error", func(t *testing.T) {
```

Comment:
this test hasn't been properly fixed, the mock is returning an error but the signature should change so that nothing is returned